src/api/spec/routing/webui/users/requests_spec.rb

```diff
@@ -1,9 +1,26 @@
 require 'rails_helper'
 
-RSpec.describe '/users/:user/requests routes' do
+RSpec.describe '/users/:user/requests routes', type: :routing do
   let(:user) { Faker::Name.first_name }
+
   it do
     expect(get("/users/#{user}/requests"))
       .to route_to('webui/users/bs_requests#index', user: user)
   end
+
```

Comment:
I don't know why did you added this route specs here in this file. This file is for BsRequest related routes, not for Http request in general...

And also I don't think we need to test the aliases for those routes. It doesn't add too much.